ibm/service/dnsservices/data_source_ibm_private_dns_custom_resolver_forwarding_rules.go

```diff
@@ -61,6 +61,36 @@ func DataSourceIBMPrivateDNSForwardingRules() *schema.Resource {
 								Type: schema.TypeString,
 							},
 						},
+						pdnsCRFRViews: {
```

Comment:
Ensure the order of views is preserved as returned by the API as the order dictates the priority of the views.

Reply:
fixed